include/sensors/core/tasks/capacitive_driver.hpp

```diff
@@ -1,7 +1,14 @@
 #pragma once
 
 #include <array>
-#include <bitset>
+
+#ifdef ENABLE_CROSS_ONLY_HEADERS
+// TODO(fps 7/12/2023): This is super hacky and I hate throwing #ifdefs
+// in our nicely host-independent code but for now we really just need
+// the vTaskDelay function and hopefully sometime in the near future I
+// can refactor this file with a nice templated sleep function.
+#include "FreeRTOS.h"
```

Comment:
Lol yuck. Let's at least make this like
```
#ifdef ENABLE_CROSS_ONLY_HEADERS
#include "FreeRTOS.h"
#define HACKY_TASK_SLEEP( ___timeout___ ) vTaskDelay(___timeout___)
#else
#define HACKY_TASK_SLEEP(___timeout___) (void)(___timeout___)
#endif
```
so we keep the ifdefs to just one place and can just do `HACKY_TASK_SLEEP(100);` later